CHANGELOG.md

```diff
@@ -2,6 +2,35 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+## Mapbox Navigation SDK 2.0.0-beta.8 - April 30, 2021
+
+ For details on how v2 differs from v1 and guidance on migrating from v1 of the Mapbox Navigation SDK for Android to the v2 public preview, see [2.0 Navigation SDK Migration Guide](https://github.com/mapbox/mapbox-navigation-android/wiki/2.0-Navigation-SDK-Migration-Guide).
+
+ ### Changelog
+ #### Features
+ * Customize voice guidance player. [#4278](https://github.com/mapbox/mapbox-navigation-android/pull/4278)
```

Comment:
This doesn't provide the data from the changelog clause in the PR (granted it would have to be refactored a little bit).